client-app/ui-kit/components/atoms/badge/vc-badge.vue

```diff
@@ -8,6 +8,8 @@
         'vc-badge--rounded': rounded,
         'vc-badge--truncate': truncate,
         'vc-badge--nowrap': nowrap,
+        'vc-badge--dot': !$slots.default,
```

Comment:
remove it please, we already have dot modifier https://github.com/VirtoCommerce/vc-frontend/pull/1750/files#diff-63dd37412e6cb03b485c2274216fe170f68be6c0804f1474220c3e837c4633f9R5